doc/manuals/taskvine/index.md

```diff
@@ -707,9 +705,9 @@ software packages and reference datasets.
 
 === "Python"
     ```python
-    f = m.declare_file("myfile.txt",cache="never")
-    f = m.declare_file("myfile.txt",cache="workflow") # (default)
-    f = m.declare_file("myfile.txt",cache="always") # (default)
+    f = m.declare_file("myfile.txt",cache="never") # (default)
```

Comment:
workflow caching is the default.

Reply:
Do we want to change the default value here then?
https://github.com/cooperative-computing-lab/cctools/blob/cdc25542f936df263b75f72e14b321ad9d5960df/taskvine/src/bindings/python3/ndcctools/taskvine/manager.py#L1355

Reply:
For now let's match the code as other code depends on this behavior. Can't think which is beneficial in the long run though..